cmd/syft/cli/options/source.go

```diff
@@ -34,7 +35,14 @@ func defaultSourceConfig() sourceConfig {
 	}
 }
 
-func (c imageSource) PostLoad() error {
+func (c *fileSource) PostLoad() error {
+	digests := strset.New(c.Digests...).List()
+	sort.Strings(digests)
+	c.Digests = digests
+	return nil
+}
+
+func (c *imageSource) PostLoad() error {
```

Comment:
Is the change of imageSource receiver from non-pointer to pointer intentional? (I'm not sure it matters; it just looks accidental.)

Reply:
let me back that out, it was unintentional (good catch)